mainform.py

```diff
@@ -1,16 +1,22 @@
-# -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*-'
+from matplotlib import rc
+rc('font',**{'family':'verdana'})
+try:
```

Comment:
¯\_(ツ)_/¯
--
Envoyé de l'application Mail.Ru pour Android vendredi, 09 septembre 2016, 03:50PM +03:00 de Cadovvl < notifications@github.com> :

>In  mainform.py :
>> @@ -1,16 +1,22 @@
> -# -*- coding: utf-8 -*-
> +# -*- coding: utf-8 -*-'
> +from matplotlib import rc
> +rc('font',**{'family':'verdana'})
> +try:
>Бессмысленная конструкция. В чем смысл оборачивания в try-except?
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .